<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Cloud</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionDetect"/>
    <addaction name="actionSegment"/>
    <addaction name="separator"/>
    <addaction name="actionPrintDetectedFaces"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShowPoints"/>
    <addaction name="actionShowFaces"/>
    <addaction name="separator"/>
    <addaction name="actionShowAll"/>
    <addaction name="actionShow1"/>
    <addaction name="actionShow2"/>
    <addaction name="actionShow3"/>
    <addaction name="actionShow4"/>
    <addaction name="actionShow5"/>
    <addaction name="actionShow6"/>
    <addaction name="actionShow7"/>
    <addaction name="actionShow8"/>
    <addaction name="actionShow9"/>
    <addaction name="actionShow10"/>
    <addaction name="actionShow11"/>
    <addaction name="actionShow12"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionRenderingBasic"/>
    <addaction name="actionRenderingSSAO"/>
    <addaction name="actionRenderingHatching"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuView"/>
   <addaction name="menuRendering"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>Resources/open.png</normaloff>Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDetect">
   <property name="text">
    <string>Detect</string>
   </property>
  </action>
  <action name="actionShowPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Points</string>
   </property>
  </action>
  <action name="actionShowFaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Faces</string>
   </property>
  </action>
  <action name="actionRenderingBasic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionRenderingSSAO">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SSAO</string>
   </property>
  </action>
  <action name="actionRenderingHatching">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hatching</string>
   </property>
  </action>
  <action name="actionSegment">
   <property name="text">
    <string>Segment</string>
   </property>
  </action>
  <action name="actionShow1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 1</string>
   </property>
  </action>
  <action name="actionShow2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 2</string>
   </property>
  </action>
  <action name="actionShow3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 3</string>
   </property>
  </action>
  <action name="actionShow4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 4</string>
   </property>
  </action>
  <action name="actionShow5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 5</string>
   </property>
  </action>
  <action name="actionShow6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 6</string>
   </property>
  </action>
  <action name="actionShow7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 7</string>
   </property>
  </action>
  <action name="actionShow8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 8</string>
   </property>
  </action>
  <action name="actionShow9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 9</string>
   </property>
  </action>
  <action name="actionShowAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show All</string>
   </property>
  </action>
  <action name="actionShow10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 10</string>
   </property>
  </action>
  <action name="actionShow11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 11</string>
   </property>
  </action>
  <action name="actionShow12">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 12</string>
   </property>
  </action>
  <action name="actionPrintDetectedFaces">
   <property name="text">
    <string>Print Detected Faces</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
